### DISCUSSION TITLE
Pybricks PID Value Range

### DISCUSSION THREAD

**Discussion by Vinz1911** [pybricks/support#435] [Q&A] [answered]
Hello again 👋

I have a question about the PID Value Ranges. In basic I think I understand how a pid controller work's and what the different values mean. But the Value ranges are a bit confusing for me. I want to tune a little bit the movement to get it a bit more smoothly. Can you maybe explain me a little bit more about the Pid Values and how "strong" they are applied? Or give me a hint in which value range it's recommended to adjust the values to make movement a bit smoother. 

Thank you and best regards :)

**Comment by laurensvalk** [comment, marked as answer]
The numbers represent, roughly, the amount of feedback torque for each degree of error.

This is a good resource to get an idea of what each motor can do: https://www.philohome.com/motors/motorcomp.htm

When it comes to tuning, though, it’s usually easier to just read the defaults for that particular motor and apply relative values, like kp = old_kp * 0.5.

**Comment by Vinz1911** [comment]
Thank you, that helped a lot :)
